### ISSUE TITLE
'See more' links in Add-on Category pages are not consistent with the links in the homepage

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/addons#11154]
Steps to reproduce:

1. Open an Extension/Theme Category page (i.e. https://addons-dev.allizom.org/en-US/firefox/extensions/alerts-updates/)
2. Observe the 'See more' links in the Featured/Top rated... shelves

Expected results:
The links have the same names as in the homepage (i.e. 'Featured extensions' shelf should have a 'See more featured extensions' link)

Actual result
The Category shelves have only a 'See more' link

Note: reproduced on all AMO servers, with FF56 (Win10 and Android 7.0)
Ref [#3633](https://github.com/mozilla/addons/issues/11020)
![image](https://user-images.githubusercontent.com/31961530/32598982-cd442162-c543-11e7-9fa7-079a5726e801.png)
